### DISCUSSION TITLE
Song select skinning / details

### DISCUSSION THREAD

**Discussion by lakazatong** [ppy/osu#27353] [Ideas] [answered]
Hi, I know skinning is still experimental on lazer, but is song select still considered?

I'd like to change the looks of the beatmaps on the right (the "cards"), and make them transparent for example
Also, something I'd love to be able to change in the settings, get rid of the "header" of the beatmap when selected
it's weird to have 2 elements selected when the beatmap only has 1 difficulty, also there is redundant information shown, all that is shown in the header is already there at the top left, maybe some love this addition, hence making it an option would be nice (it also makes the actual difficulties tiny tiny, while being the main elements that we are selecting)

Moreover, the search bar is huge and takes quite an amount of place, I'd like to make it thinner, smaller, move it maybe, detach it from the top bar, or make the search bar invisible when no text is entered
Also, the fact that we now need two clicks to change collection feels weird as collections are an essential part of song select, whereas the "Difficulty range" is never used by some players, so making it an always accessible element and as big as it is is unnecessary (tho having it there by default is great for newer players, showing them what's possible)

And finally, the "..." for the "Sort by" section shows filters that are already shown, it would be better to only show the remaining filters and make it dynamic with screen size for players that does not play in fullscreen or other screen ratios, and make it customizable as well, one could want to have "Last played" first

Song select and Gameplay are the two most important sections of the game, this is where players will spend most of their time, if it's not waiting in a multiplayer lobby for someone to pick a map, so I'd love to see more customization for Song select, as Gameplay now feels and looks perfect to me

Overall, Song select feels static and not customizable in contrast to Gameplay while in the skin editor

As for the wiki, osu lazer lack one, especially on what's customizable, I've seen a video from peppy importing (by drag and dropping) an image to override a texture, but I'd like to have some place that tells me exactly what filename overwrites what, or an assets library folder that will show its content on the left (components) while in the skin editor, and maybe have folders show up as a selectable element in a dropdown menu so that players can organize their assets themselves

**Comment by Rekunan** [comment, marked as answer]
Yes, song select will be skinnable in the future.
